react-app/src/Pages/Register Page/RegisterPage.js

```diff
@@ -0,0 +1,58 @@
+import './RegisterPage.css';
+import NavBar from '../../Components/navBar/navBar';
+import { TextField, Typography, Button } from '@mui/material';
+import { useState } from 'react';
+import axios from 'axios';
+
+function LoginPage() {
+    const [username, setUsername] = useState('')
+    const [password, setPassword] = useState('')
+
+    const submitDetails = (e) => {
+        e.preventDefault()
+
+        if (username && password) {
+            console.log(username, password)
+        }
+        else {
+            console.log('error')
+        }
+    }
+
+    const registerDetails = async(e) => {
```

Comment:
Try catch on different branch?

Reply:
This is an older version of the login page that I duplicated for the register page.